### DISCUSSION TITLE
Repository not linked in dependency graph

### DISCUSSION THREAD

**Discussion by makupi** [community/community#22092] [New to GitHub]
<p>I have a pypi package that is on github <a href="https://github.com/makupi/async-nookipedia" rel="nofollow noopener">here</a>, and another application that is using it <a href="https://github.com/makupi/acnh-bot/" rel="nofollow noopener">here</a>. The application shows the pypi package properly under dependencies but it’s grayed out, for the pypi package the depentants graph shows no repositories at all - how do I fix that?<br>
Thanks in advance!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/makupi">@makupi</a>, Welcome to the community! We have an existing request to support the python package manager (pip) currently sitting with our Product team for consideration and exploration; I’ve added your request as a  <code>+1</code>  to this.</p>
<p>We don’t currently have an ETA on this, however you can keep an eye on the<br>
<a href="https://github.blog/changelog/">GitHub Changelog</a> for future announcements on new features being added to GitHub Packages.</p>
